s3tests_boto3/functional/test_s3_neofs.py

```diff
@@ -598,3 +598,34 @@ def test_multipart_upload():
     _check_content_using_range(key, bucket_name, data, 1000000)
     _check_content_using_range(key, bucket_name, data, 10000000)
     client.delete_object(Bucket=bucket_name, Key=key)
+
+
+@attr(resource='object')
+@attr(operation='object workflow')
```

Comment:
The same as in https://github.com/nspcc-dev/s3-tests/pull/20#discussion_r892810185